prefs.js

```diff
@@ -112,9 +112,29 @@ function buildPrefsWidget() {
   );
   prefsWidget.attach(feedbackSoundsSwitch, 1, 4, 1, 1);
 
+  const playingSoundNotSupportedLabel = new Gtk.Label({
+    halign: Gtk.Align.START,
+    visible: !is_playing_sound_supported(),
+  });
+  playingSoundNotSupportedLabel.set_markup("<span foreground='red'>WARNING. Playing sound is not supported on this system. Is GStreamer package installed?</span>");
+  prefsWidget.attach(playingSoundNotSupportedLabel, 0, 5, 1, 1);
```

Comment:
Add `playingSoundNotSupportedLabel.set_line_wrap(true)` to wrap the text. Otherwise it forces the window to be wider than it needs to be.

Reply:
I tried (with Gnome Shell 40.9) and it fails with:
> TypeError: playingSoundNotSupportedLabel.set_line_wrap is not a function

It seems to be available only in the Python bindings. In JS, I had to use `set_wrap(true)`.

Reply:
Looks like this is another case where we need to check Gnome shell version. I get:
`TypeError: playingSoundNotSupportedLabel.set_wrap is not a function`
on 3.36.9😞.

Reply:
Maybe that's also why `set_visible()` doesn't do anything for me? The warning still shows, even if I explicitly do `set_visible(false)`.

Reply:
Could you paste that section in the way that works also with 3.36? Maybe we will be able to do some sensible conditional work?

Reply:
Wrapping the whole label code (line 117 - 123) in `    if(!isPlayingSoundSupported)` solves visibility. Adding
```
if (GTK_VERSION == 3) {
  playingSoundNotSupportedLabel.set_line_wrap(true);
} else {
  playingSoundNotSupportedLabel.set_wrap(true);
}
```
solves the wrapping. Note again that this is with Gnome shell 3.36.9, so it needs to be tested with other versions.

Reply:
The whole thing would look like this:
```
  if(!isPlayingSoundSupported) {
    const playingSoundNotSupportedLabel = new Gtk.Label({
      halign: Gtk.Align.START,
    });
    playingSoundNotSupportedLabel.set_markup("<span foreground='red'>WARNING. Playing sound is not supported on this system. Is GStreamer package installed?</span>");
    if (GTK_VERSION == 3) {
      playingSoundNotSupportedLabel.set_line_wrap(true);
    } else {
      playingSoundNotSupportedLabel.set_wrap(true);
    }
    prefsWidget.attach(playingSoundNotSupportedLabel, 0, 5, 1, 1);
  }
```

Reply:
Cool, thanks. It works also with 40.9.

Reply:
👍